ament_clang_tidy/ament_clang_tidy/main.py

```diff
@@ -277,9 +277,12 @@ def get_compilation_db_files(paths):
     for path in paths:
         if os.path.isdir(path):
             for dirpath, dirnames, filenames in os.walk(path):
-                if 'AMENT_IGNORE' in dirnames + filenames:
-                    dirnames[:] = []
-                    continue
+                # NOTE: here we don't check for the AMENT_IGNORE file.
+                # This function tries to find compile_commands.json file in the build folders.
+                # Build folders always include a AMENT_IGNORE file, so checking for it would result in not finding any compilation db.
+                # The check would also be redundant, because if a source folder was marked with AMENT_IGNORE, then 
```

Comment:
Actually, flake8 thinks these lines are too long (they need to be < 100).  Once that is fixed we can run full CI on it.